src/panels/lovelace/common/directives/long-press-directive.ts

```diff
@@ -136,8 +136,16 @@ class LongPress extends HTMLElement implements LongPress {
     element.addEventListener("touchstart", clickStart, { passive: true });
     element.addEventListener("touchend", clickEnd);
     element.addEventListener("touchcancel", clickEnd);
-    element.addEventListener("mousedown", clickStart, { passive: true });
-    element.addEventListener("click", clickEnd);
+
+    // iOS 13 sends a complete normal touchstart-touchend series of events followed by a mousedown-click series.
+    // That might be a bug, but until it's fixed, this should make long-press work.
+    // If it's not a bug that is fixed, this might need updating with the next iOS version.
+    // Note that all events (both touch and mouse) must be listened for in order to work on computers with both mouse and touchscreen.
+    const isIOS13 = window.navigator.userAgent.match(/iPhone OS 13_/);
```

Comment:
we should match anything that runs iOS, so ipad, iphone and ipod ? I also think that we might want to match any version >= 13

Reply:
Something like this https://github.com/lancedikson/bowser/blob/642fd6164f7cbef1198de4029d0e5bf396aae034/src/parser-os.js#L62-L73